CHANGELOG.md

```diff
@@ -3,6 +3,57 @@
 All notable changes to this project will be documented in this file.
 See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
 
+## 6.0.2
+
+_Mar 16, 2023_
+
+We'd like to offer a big thanks to the 10 contributors who made this release possible. Here are some highlights ✨:
+
+- 🚀 Fire `onChange` when filling a partial date (#8082) @flaviendelangle
+- 🌍 Add Hebrew (he-IL) locale (#8222) @ylarom
+- 🌍 Improve Brazilian Portuguese (pt-BR), German (de-DE), and French (fr-FR) locales
+- 📚 Documentation improvements
+- 🐞 Bugfixes
+
+### `@mui/x-data-grid@v6.0.2` / `@mui/x-data-grid-pro@v6.0.2` / `@mui/x-data-grid-premium@v6.0.2`
+
+#### Changes
+
+- [DataGrid] Fix <kbd>Space</kbd> triggering edit mode (#8180) @m4theushw
+- [l10n] Improve Brazilian Portuguese (pt-BR) locale (#8198) @JoaoSerafim3001
+
+### `@mui/x-date-pickers@v6.0.2` / `@mui/x-date-pickers-pro@v6.0.2`
+
+#### Changes
+
+- [l10n] Add Hebrew (he-IL) locale (#8222) @ylarom
+- [l10n] Improve German (de-DE) locale (#8204) @sebkasanzew
+- [l10n] Improve French (fr-FR) locale (#8229) @marvinroger
+- [DateRangePicker] Allow overriding `slotProps.textField` (#8201) @LukasTy
+- [fields] Fire `onChange` when filling a partial date (#8082) @flaviendelangle
+- [fields] Fix editing in shadow dom (#8254) @flaviendelangle
+- [fields] Remove the warning about invalid adapter (#8187) @flaviendelangle
```

Comment:
With the current form, one might think that we don't warn at all, which would be weird

```suggestion
- [fields] Remove the duplicated warning about invalid adapter (#8187) @flaviendelangle
```